examples/r_examples/lasso_api.R

```diff
@@ -0,0 +1,29 @@
+# lasso_api.R
+
+# Global code; gets executed at plumb() time.
+if(!require(glmnet)){
+  install.packages("glmnet")
+}
+library(glmnet)
+data(longley)
```

Comment:
@swilliamson-cs Where does this data actually come from (as in how is it loaded)?  If we want to be able to perform Certifai analysis we need it as explicit CSVs (also ideally the models would train from provided CSVs, but that point is perhaps more finessable).  Similarly the other datasets

Reply:
I've changed it to use German Credit and Auto Insurance as appropriate, and cleaned up the implementations a little.